numpyro/infer/svi.py

```diff
@@ -146,7 +146,25 @@ def update(self, svi_state, *args, **kwargs):
         loss_val, optim_state = self.optim.eval_and_update(loss_fn, svi_state.optim_state)
         return SVIState(optim_state, rng_key), loss_val
 
-    def run(self, rng_key, num_steps, *args, progress_bar=True, **kwargs):
+    def stable_update(self, svi_state, *args, **kwargs):
+        """
+        Similar to :meth:`update` but returning the current state if the
```

Comment:
nit: Similar to :meth:`update` but returns the current state if the
        the loss or the new state contains invalid values.